XrmToolBox/Forms/PluginsChecker.cs

```diff
@@ -141,6 +83,84 @@ public void RefreshPluginsList()
             bw.RunWorkerAsync();
         }
 
+        private ListViewItem AddPackage(IPackage package)
+        {
+            var xtbPackage = new XtbNuGetPackage(package, PackageInstallAction.None);
+            var item = new ListViewItem(xtbPackage.ToString());
+            item.Tag = xtbPackage;
+            item.SubItems.Add(package.Version.ToString());
+            var currentVerItem = item.SubItems.Add("");  //Current version
+            item.SubItems.Add(package.Description);
+            item.SubItems.Add(string.Join(", ", package.Authors));
+            var actionItem = item.SubItems.Add("None");
+
+            var files = package.GetFiles();
+
+            bool install = false, update = false, compatible = false;
+
+            var xtbDependency = package.FindDependency("XrmToolBox", null);
+            if (xtbDependency != null)
+            {
+                var xtbDependencyVersion = xtbDependency.VersionSpec.MinVersion.Version;
+                compatible = IsPluginDependencyCompatible(xtbDependencyVersion);
+            }
+
+            var currentVersion = new Version(int.MaxValue, int.MaxValue, int.MaxValue, int.MaxValue);
+            var currentVersionFound = false;
+
+            // TODO: Don't compare with all files, plugin packages may contain other dll's and exe's that have other versioning

```

Comment:
correct... I didn't read the code well...

Maybe we can add a txt file in the package that contains the nuget package version so that we can  compare with this file...